framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java

```diff
@@ -52,6 +52,7 @@
   private TestMatrix matrix;
   private Thread thread;
   private List<Integer> types;
+  private List<Integer> nullability;
```

Comment:
Are you suggesting we track types and nullability differently? Currently they are lumped together. I don't feel it's necessary to separate them out. It's very clear by looking at the output what is failing the test.